src/app/login/page.tsx

```diff
@@ -1,92 +1,77 @@
-"use client"
-import Cookie from 'js-cookie'
-import{useRouter} from 'next/navigation'
-import { FormEvent } from 'react'
+"use client";
+import React, { useState } from "react";
+import Cookie from "js-cookie";
+import { useRouter } from "next/navigation";
+import { FormEvent } from "react";
 
-export default function Home() {
-    const router = useRouter()
-    
-    function handleLogin(){
-        Cookie.set("auth_token", "aisodnaoisndauisndbiausbnda")
-        router.push("/")
-        console.log(Cookie.get('auth_token'))
-    }
+const LoginUsuario: React.FC = () => {
```

Comment:
React functional component não é necessario
```suggestion
const Login = () => {
```